tsdb/engine/tsm1/engine.go

```diff
@@ -2047,15 +2047,14 @@ func (e *Engine) compact(wg *sync.WaitGroup) {
 	}
 }
 
-// compactHiPriorityLevel kicks off compactions using the high priority policy. It returns
+// compactLevel kicks off compactions using the level strategy. It returns
 // true if the compaction was started
-func (e *Engine) compactHiPriorityLevel(grp CompactionGroup, level int, fast bool, wg *sync.WaitGroup) bool {
+func (e *Engine) compactLevel(grp CompactionGroup, level int, fast bool, wg *sync.WaitGroup) bool {
 	s := e.levelCompactionStrategy(grp, fast, level)
```

Comment:
Method compactLoPriorityLevel and method compactHiPriorityLevel code are exactly the same.